docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -11,19 +11,21 @@
    - To check your Windows version, run `winver` on the command line.
 
 ## Software prerequisites
+For specific versions we use for each release, see [install-windows-prereqs.ps1](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1#L6-L36).
```

Comment:
Pointing into code is not good docs.  Don't assume people know how to read/interpret the data in that link.

Reply:
Thanks Dave. I did make an assumption here. I will remove this line. Do you think it would be appropriate to point to a markdown file containing a list of dependencies? For example `Windows Server 2019 prerequisites.md` with the contents
>
# Windows Server 2019 OpenEnclave SDK Prerequisites
[Git v2.31.1](https://github.com/git-for-windows/git/releases/download/v2.31.1.windows.1/Git-2.31.1-64-bit.exe)
[OpenSSL 1.1.1j](https://oejenkins.blob.core.windows.net/oejenkins/openssl.1.1.1506.73.nupkg)
[SevenZip](https://www.7-zip.org/a/7z1806-x64.msi)
[VSBuildTools](https://aka.ms/vs/15/release/vs_buildtools.exe)
[Clang 10.0.0](https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe)
[Intel SGX PSW v2.13.100.2](https://registrationcenter-download.intel.com/akdlm/irc_nas/18000/Intel%20SGX%20PSW%20for%20Windows%20v2.13.100.2.exe)
[ShellCheck v0.7.0](https://oejenkins.blob.core.windows.net/oejenkins/shellcheck-v0.7.0.zip)
[Nuget 3.4.3](https://www.nuget.org/api/v2/package/NuGet.exe/3.4.3)
[Devcon](https://download.microsoft.com/download/7/D/D/7DD48DE6-8BDA-47C0-854A-539A800FAA90/wdk/Installers/787bee96dbd26371076b37b13c405890.cab)
[Intel SGX DCAP v1.11.100.2](https://registrationcenter-download.intel.com/akdlm/irc_nas/17999/Intel%20SGX%20DCAP%20for%20Windows%20v1.11.100.2.exe)
[VC Runtime 2012](https://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe)
[Azure DCAP 1.10.0](https://www.nuget.org/api/v2/package/Microsoft.Azure.DCAP/1.10.0)
[Python3](https://www.python.org/ftp/python/3.7.4/python-3.7.4-embed-amd64.zip)
[NSIS](https://oejenkins.blob.core.windows.net/oejenkins/nsis-3.05-setup.exe)
[Python pip](https://bootstrap.pypa.io/pip/3.4/get-pip.py)

Reply:
@CyanDevs, I think having a markdown is a great way to keep things up to date.